tests/Resource/OtpTest.php

```diff
@@ -107,62 +86,59 @@ public function testCancelById(): void
         }
     }
 
-    public function testGetById(): void
+    public function testCancelByDestinationNotFound(): void
     {
-        $responseBody = '{"requestId":"404372541682577504482079","validUnitlTimestamp":"2020-11-06 13:59:11","createdTimestamp":"2020-11-06 13:49:11","lastEventTimestamp":"2020-11-06 13:49:33","status":"Verified"}';
+        $this->expectException(ResourceNotFoundException::class);
 
         // Create a mock and queue two responses.
         $mock = new MockHandler([
-            new Response(200, ['content-type' => 'application/json'], $responseBody),
+            new Response(404),
         ]);
         $handlerStack = HandlerStack::create($mock);
         $client = new Client(['handler' => $handlerStack]);
-
         try {
             $otp = new Otp($client);
-            $otpResponse = $otp->getById('404372541682577504482079');
-            $this->assertEquals('404372541682577504482079', $otpResponse['requestId']);
-        } catch (GuzzleException $e) {
-            $this->fail('This test should not have failed');
+            $otp->cancelByDestination('404372541682577504482079');
         } catch (CredentialsException $e) {
             $this->fail('This test should not have failed');
-        } catch (AuthenticationException $e) {
-            $this->fail('This test should not have failed');
-        } catch (InvalidResponseException $e) {
+        } catch (GuzzleException $e) {
             $this->fail('This test should not have failed');
-        } catch (ResourceNotFoundException $e) {
+        } catch (AuthenticationException $e) {
             $this->fail('This test should not have failed');
         }
     }
 
-    public function testGetByIdNotFound(): void
+    public function testCancelByDestination(): void
     {
-        $this->expectException(ResourceNotFoundException::class);
+        $responseBody = '{"requestId":"404372541682577504482079","destination": "61474950859", "validUnitlTimestamp":"2020-11-06 13:59:11","createdTimestamp":"2020-11-06 13:49:11","lastEventTimestamp":"2020-11-06 13:49:33","status":"Cancelled"}';
 
         // Create a mock and queue two responses.
         $mock = new MockHandler([
-            new Response(404),
+            new Response(200, ['content-type' => 'application/json'], $responseBody),
```

Comment:
Use a HTTP status ok constant

Reply:
PHP/Guzzle does not have such constants. I dont mind adding a trait but then we have to look further for code coverage report.